src/Rules.UnitTests/EmbeddedResourceTests.cs

```diff
@@ -31,6 +31,8 @@ namespace SonarLint.VisualStudio.Rules.UnitTests
     [TestClass]
     public class EmbeddedResourceTests
     {
+        private const string BaseResourcePath = "SonarLint.VisualStudio.Rules.Embedded.";
```

Comment:
Miscellaneous: moved the constant to the top of the file, which is where are by convention